### PR TITLE
Update OTel runtime metrics documentation

### PR DESCRIPTION
**Pull request by liustanley**
### What does this PR do?

- Updates Java runtime metric mapping
- Adds important notes to .NET documentation

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/opentelemetry/runtime_metrics/dotnet.md [hunk 1]
```diff
@@ -10,13 +10,17 @@ code_lang_weight: 20
 
 - You are successfully [sending OpenTelemetry metrics to Datadog][1].
 - You have installed the language integration for [.NET][2].
+- Minimum .NET OTel SDK version of [1.5.0][6]
```
Comment (drichards-87) [inline content/en/opentelemetry/runtime_metrics/dotnet.md:13]:
```suggestion
- Minimum .NET OTel SDK version of [1.5.0][6].
```

File: content/en/opentelemetry/runtime_metrics/dotnet.md [hunk 2]
```diff
@@ -10,13 +10,17 @@ code_lang_weight: 20
...
 
 ## OpenTelemetry SDK configuration
 
 If you have instrumented your .NET applications with [OpenTelemetry automatic instrumentation][3], runtime metrics are automatically enabled. 
 
 If you have instrumented your .NET application with [OpenTelemetry manual instrumentation][4], see the documentation for the [OpenTelemetry.Instrumentation.Runtime library][5].
 
+The default metric export interval for the .NET OTel SDK is different from the default for the Datadog .NET SDK. We recommend setting the [OTEL_METRIC_EXPORT_INTERVAL][7] environment variable on your .NET service to match the default Datadog metric export interval for viewing integration metric graphs:
```
Comment (drichards-87) [inline content/en/opentelemetry/runtime_metrics/dotnet.md:21]:
```suggestion
The default metric export interval for the .NET OTel SDK is different from the default for the Datadog .NET SDK. Datadog recommends setting the [OTEL_METRIC_EXPORT_INTERVAL][7] environment variable on your .NET service to match the default Datadog metric export interval for viewing integration metric graphs:
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you two very minor bits of feedback from Docs.

**Review by drichards-87 — APPROVED** [review]

**Comment by drichards-87** [comment]
@liustanley Is this ready to be merged?

**Comment by liustanley** [comment]
@drichards-87 I think we can hold off until Agent 7.48.0 gets released, this is the PR with the change: https://github.com/DataDog/datadog-agent/pull/18587

**Comment by drichards-87** [comment]
Hi @liustanley. Do you still need me to merge this?

**Comment by liustanley** [comment]
@drichards-87 Yes but we're still waiting for Agent 7.48.0 to be released, which is targeted for mid September

**Comment by liustanley** [comment]
@drichards-87 Agent 7.48.0 was just released, could you merge this PR?

**Pull request merged**